game_template/lib/src/settings/settings_screen.dart

```diff
@@ -169,14 +169,19 @@ class _SettingsLine extends StatelessWidget {
       child: Padding(
         padding: const EdgeInsets.symmetric(horizontal: 8),
         child: Row(
-          mainAxisAlignment: MainAxisAlignment.center,
+          mainAxisAlignment: MainAxisAlignment.start,
           children: [
-            Text(title,
+            Expanded(
+              child: Text(
+                title,
+                maxLines: 1,
+                overflow: TextOverflow.ellipsis,
                 style: const TextStyle(
                   fontFamily: 'Permanent Marker',
                   fontSize: 30,
-                )),
-            const Spacer(),
+                ),
+              ),
+            ),
```

Comment:
Allows text to overflow when the screen is too small, showing an ellipsis `...` instead. This was necessary to pass the test after moving to Material 3.